src/content/HelmetContent.tsx

```diff
@@ -1,9 +1,11 @@
-import SvgLogoWhite from '../images/logo_white.svg';
 import { landing } from './LandingContent';
 
 export const helmet = {
   description: landing.presentation, // This should not be too long (160 chars)  see https://www.gatsbyjs.com/tutorial/seo-and-social-sharing-cards-tutorial/#querying-with-graphql
-  profile: {
-    logo: { src: SvgLogoWhite },
+  logo: {
+    src: '/image/twitter_card.webp',
```

Comment:
⚠️ property name is not accurate: this is for usage in `og:image`
Same remarks for the image file name

Reply:
Done